packages/server/postgres/migrations/1720689742934_fixThreatLevelPromptColors.ts

```diff
@@ -0,0 +1,42 @@
+import {r} from 'rethinkdb-ts'
+import connectRethinkDB from '../../database/connectRethinkDB'
+
+export async function up() {
+  await connectRethinkDB()
+
+  await r
+    .table('ReflectPrompt')
+    .filter({id: 'threatLevelPremortemTemplateSeverePrompt'})
```

Comment:
-1 use `get` instead of `filter`, otherwise it won't use the index

Reply:
Done and tested locally again both migrating up and down! Thanks for the correction!